missing fields in PH address format

TagaSanPedroAko opened the pull request:
resolves #9680

TagaSanPedroAko committed:
missing fields in PH address format

tyrasd reviewed (commented):
Thank you and please excuse the delayed review.

This seems to be mostly in line with https://wiki.openstreetmap.org/wiki/Philippines/Addressing#Proposed_schema (and [usage](http://taginfo.geofabrik.de/asia/philippines/search?q=addr%3A)), but is using `addr:district` which is not mentioned on the page (maybe this was meant as an alternative to `addr:barangay`?). I'm ok with supporting either schema, but the wiki should at least mention the use of `addr:district`.

Another, more UI related question: I'd say that putting the `addr:housename` field in the same line as the house number and street would make the individual fields too narrow to be easily readable. Would it not be better to put the `housename` field in an extra line above or below the "street line"?

tyrasd committed:
switch to "addr:barangay"

As documented in https://wiki.openstreetmap.org/wiki/Philippines/Addressing#Proposed_schema.

Also usage seems to be indicate this is the more commonly used tag: http://taginfo.geofabrik.de/asia:philippines/keys/addr%3Abarangay#chronology

tyrasd committed:
replace non-standard addr:barangay with addr:district
